src/App.css

```diff
@@ -20,92 +21,96 @@ h2,
 h3,
 h4,
 p {
-    margin: 0;
+  margin: 0;
 }
 
 main {
-    text-align: center;
+  text-align: center;
 }
 
 li {
-    list-style: none;
+  list-style: none;
 }
 
 .nav-container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
 nav {
-    position: absolute;
-    bottom: 2rem;
+  position: absolute;
+  bottom: 2rem;
 }
 
 a {
-    text-decoration: none;
-    margin: 1rem;
+  text-decoration: none;
+  margin: 1rem;
 }
 
-    a:visited {
-        color: black;
-    }
+a:visited {
+  color: black;
+}
 
 .active {
-    font-weight: bold;
+  font-weight: bold;
 }
 
 .button-home {
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
-    padding: 0.25em 0.75em;
-    background-color: #e0d4f6;
-    box-shadow: 0 3px 5px rgba(0, 0, 0, 0.18);
-    font-family: inherit;
-    font-size: 1.5rem;
-    line-height: 1.1;
-    text-align: center;
-    cursor: pointer;
-    transition: 220ms all ease-in-out;
-    color: #675883;
-    font-weight: bold;
-    border-radius: 0.15em;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.25em 0.75em;
+  background-color: #e0d4f6;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.18);
+  font-family: inherit;
+  font-size: 1.5rem;
+  line-height: 1.1;
+  text-align: center;
+  cursor: pointer;
+  transition: 220ms all ease-in-out;
+  color: #675883;
+  font-weight: bold;
+  border-radius: 0.15em;
 }
 
 .button__home:hover,
 .button__home:active {
-    background-color: powderblue;
```

Comment:
That's a mistake from a previous PR. Somebody probably changed the class in the jsx but forgot to change it in the CSS. I'll fix it though, thanks.

Reply:
The error you're getting is because the collection we're writing to in this PR has been changed (Katia says she discussed that with you). This code queries the `lists` collection to verify the user input, but if you generate a token and add an item, it still gets added to the `shoppinglist` collection, because the previous PR writes to the `shoppinglist` collection. So the new token doesn't token doesn't exist in the `list` collection, which is why we get that error.

That difference will be fixed when the other team merges their work. If you generate the token [here](https://tcl-25-shopping-list--pr21-ku-ci-prevent-duplic-x9wpcah1.web.app/) (the other team's link), and then enter the token in our deployed version, everything should work fine.
Sorry for the inconvenience, and thank you!

Reply:
Thanks for the detailed explanation, everything works fine :+1:

Reply:
Thank you. Can we go ahead and merge now?

Reply:
yeah sure